core/src/test/java/io/questdb/griffin/engine/functions/regex/ILikeFunctionFactoryTest.java

```diff
@@ -260,6 +260,60 @@ public void testLikeStringUnderscoreAtStartAndEnd() throws Exception {
         });
     }
 
+
+    // JANE
+    @Test
+    public void testLikeEscapePercengtage() throws Exception {
+        assertMemoryLeak(() -> {
+            /**
+             * The string stored in the table is
+             * Can you give me 30% discount?
+             * The pattern I query is
+             * %30\%%
+             */
+
+
```

Comment:
casting to string is unnecessary .
Only the first union-ed subquery needs column aliases .

Reply:
You can simplify commands by creating table separately and inserting data with 
```sql
insert into myTable values (string1), (string2), string(3); 
```